src/Collection/Iterator/FilterIterator.php

```diff
@@ -43,7 +43,7 @@ class FilterIterator extends Collection
      * in the current iteration, the key of the element and the passed $items iterator
      * as arguments, in that order.
      *
-     * @param \Iterator $items The items to be filtered.
+     * @param \Iterator|\Traversable|array $items The items to be filtered.
```

Comment:
`Iterator` extends `Traversable`, so have both is redundant. `\Iterator` can be removed now.